cms/templates/settings.html

```diff
@@ -419,11 +419,20 @@ <h3 id="heading-entrance-exam">${_("Entrance Exam")}</h3>
                 <h2 class="title-2">${_("Course Pacing")}</h2>
                 <span class="tip">${_("Set the pacing for this course")}</span>
               </header>
+              <span class="msg" id="course-pace-toggle-tip"></span>
 
-              <input type="radio" name="self-paced" id="course-pace-self-paced" value="true"/>
-              <label for="course-pace-self-paced">Self-Paced</label>
-              <input type="radio" name="self-paced" id="course-pace-instructor-led" value="false"/>
-              <label for="course-pace-instructor-led">Instructor Led</label>
+              <ol class="list-input">
+                <li class="field">
+                  <input type="radio" class="field-radio" name="self-paced" id="course-pace-instructor-led" value="false"/>
+                  <label for="course-pace-instructor-led">Instructor Led</label>
+                  <span class="tip">${_("Instructor-led courses progress at a pace set by the course author. You can configure release dates for course content, and due dates for assignments.")}</span>
```

Comment:
Instructor-led courses progress at the pace that the course author sets. You can configure release dates for course content and due dates for assignments.